testing/functional/helpers/testHelper.ts

```diff
@@ -1,6 +1,6 @@
 import { ClientFunction } from 'testcafe';
 
-export async function createWidget(
+export default async function createWidget(
```

Comment:
Why? It is illogical that module 'helpers/testHelper' has default import createWidget.
I think that testHelper should be renamed to 'helpers/create_widget' or createWidget should not be default export.